src/common/config_opts.h

```diff
@@ -275,7 +275,9 @@ OPTION(mon_osd_auto_mark_new_in, OPT_BOOL, true)      // mark booting new osds '
 OPTION(mon_osd_down_out_interval, OPT_INT, 600) // seconds
 OPTION(mon_osd_down_out_subtree_limit, OPT_STR, "rack")   // smallest crush unit/type that we will not automatically mark out
 OPTION(mon_osd_min_up_ratio, OPT_DOUBLE, .3)    // min osds required to be up to mark things down
-OPTION(mon_osd_min_in_ratio, OPT_DOUBLE, .3)   // min osds required to be in to mark things out
+OPTION(mon_osd_max_small_cluster, OPT_INT, 10)   // Maximum number of osds that constitutes a small cluster for min in ratio
+OPTION(mon_osd_small_min_in_ratio, OPT_DOUBLE, .30)   // min osds required to be in to mark things out in small cluster
+OPTION(mon_osd_min_in_ratio, OPT_DOUBLE, .75)   // min osds required to be in to mark things out
```

Comment:
@ktdreyer The code doesn't look like that anymore.  I commented on it because I did have adjust the default for the config that did change.  Look at e8fd3e215150f302521bb93b846e71b5f07da4b6